book/website/collaboration/ppie-events/ppie-events-tech.md

```diff
@@ -0,0 +1,100 @@
+
+The flexibility of hybrid formats recognizes the barriers faced by many people with lived experience. 
+Providing a virtual option can overcome any physical resource barriers presented by in-person only events. 
+This flexibility also accommodates the diverse schedules and responsibilities of potential attendees, and takes steps to acknowledge any additional scheduling challenges that may be faced by people with lived experiences.
+
+Although planning and hosting hybrid events often requires more resources to be successful, by making your event hybrid you can better prioritize accessibility practices and the inclusion of attendees with lived experience.
+This empowers a wide range of participants to contribute and benefit from your event's content and community.
+
+If you are planning a hybrid event, please read The Turing Way's {ref}`Chapter on Hybrid Collaboration<cl-hybrid-collab> and specifically the {ref}`Subchapter on Guidelines for Hybrid Collaboration<cl-hybrid-collab-guidelines>`.
+
+(cl-ppie-events-calendar-invites)=
+## Using Email Reminders and Calendar Invitations
+Many people with lived experience have to maintain a very busy schedule and often rely on online calendar tools and email reminders to stay organized. 
+
+Consider sending both a calendar placeholder invitation for your event, and email reminders the week before, the week of, and the morning of your event. 
+This can help remind your attendees of their registration and help put the important information about your event at the top of their email inbox.
+
+It can also be helpful to utilize the subject line of your email with important information. 
+Consider including the date and title of your event, in both the calendar invitation and any email reminders that you send. 
+
+```{note}
+The Public and Patient Involvement and Engagement community for Multiple Long-Term Conditions has highlighted the importance of sending out reminder emails the week of the event, and the morning of the event. 
+
+This was because medical appointments had often been scheduled after an event's "Save the Date" had been sent and many community members valued the reminders. 
+```
+
+Ensure that any email address you send invitations from is able to contact your attendees (and not flagged as a spam email account). 
+You may suggest that attendees add your event's core email address to a list of trusted senders, or you can also ask for feedback from attendees leading up to an event, which allows you to identify if your emails have been received. 
+
+### Reaching attendees without email access
+Your event may engage communities and people with lived experience who have barriers to accessing email or the internet regularly. 
+
+Consider how you will keep in contact with these individuals, and what additional resources and safeguarding your organizers and staff members may need. 
```

Comment:
Do these events usually have safeguarding protocols? I've seen some official plans, but not sure how it usually is in a PPIE context. Ignore me if not!